composer.json

```diff
@@ -15,7 +15,9 @@
         "rapidez/account": "~0.10",
         "rapidez/core": "~0.55",
         "rapidez/image-resizer": "~0.5",
-        "rapidez/menu": "~0.2"
+        "rapidez/menu": "~0.2",
+        "rapidez/reviews": "~0.5",
+        "rapidez/wishlist": "~0.3"
```

Comment:
So because we require these by default we need to change the docs, they're not optional but installed by default now: https://docs.rapidez.io/1.x/packages.html#base